src/types.ts

```diff
@@ -156,3 +156,16 @@ export interface Addon {
   declaration?: (this: UnimportContext, dts: string, options: TypeDeclarationOptions) => Thenable<string>
   matchImports?: (this: UnimportContext, identifiers: Set<string>, matched: Import[]) => Thenable<Import[] | void>
 }
+
+export interface InstallGlobalOptions {
+  /**
+   * @default globalThis
+   */
+  globalObject?: any
+
+  /**
+   * Overrides the existing property
+   * @default false
+   */
+  overrides?: boolean
```

Comment:
```suggestion
  overrideGlobal?: boolean
```

Reply:
I am not sure if I got the semantic, you are trying to improve, can you explain a bit more? I feel the `global` is already presented in the function name and changing this creates a little duplication to me personally.